prac_05/wimbledon.py

```diff
@@ -0,0 +1,49 @@
+"""
+CP1404 Practical 5
+Wimbledon
+"""
+
+FILENAME = "wimbledon.csv"
+COUNTRY_INDEX = 1
+CHAMPION_INDEX = 2
+
+
+def main():
+    records = get_past_records(FILENAME)
+    champion_to_count, countries = create_records(records)
+    display_records(champion_to_count, countries)
+
+
+def get_past_records(file):
+    records = []
+    with open(file, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+def create_records(records):
+    champion_to_count = {}
+    countries = []
+    for record in records:
+        countries.append(record[COUNTRY_INDEX])
+        try:
+            champion_to_count[record[CHAMPION_INDEX]] += 1
+        except KeyError:
+            champion_to_count[record[CHAMPION_INDEX]] = 1
+    return champion_to_count, countries
```

Comment:
You could make countries a set here if you like by returning `set(countries)`. This way, you wouldn't have to have `countries` and `country_winners`